queries/analytics/pageview/getPageviewMetrics.js

```diff
@@ -34,26 +34,20 @@ async function relationalQuery(website_id, start_at, end_at, column, table, filt
   );
 }
 
-async function clickhouseQuery(website_id, start_at, end_at, column, table, filters = {}) {
+async function clickhouseQuery(website_id, { startDate, endDate, column, filters = {} }) {
   const { rawQuery, parseFilters, getBetweenDates } = clickhouse;
   const params = [website_id];
-  const { pageviewQuery, sessionQuery, eventQuery, joinSession } = parseFilters(
-    table,
-    column,
-    filters,
-    params,
-    'session_uuid',
-  );
+  const { pageviewQuery, sessionQuery, eventQuery } = parseFilters(column, filters, params);
 
   return rawQuery(
     `select ${column} x, count(*) y
-    from ${table}
-      ${joinSession}
-    where ${table}.website_id= $1
-    and ${getBetweenDates(table + '.created_at', start_at, end_at)}
-    ${pageviewQuery}
-    ${joinSession && sessionQuery}
-    ${eventQuery}
+    from event
+    where website_id= $1
+      ${column !== 'event_name' ? `and event_name = ''` : ''}
+      and ${getBetweenDates('created_at', startDate, endDate)}
+      ${pageviewQuery}
+      ${sessionQuery}
```

Comment:
How does this work without knowing you need to join session?

Reply:
joinSession only needs to occur if data is collected outside of the primary table being collected on.  Now that everything is merged into 1 table, a join never needs to occur.